errs/errs.go

```diff
@@ -0,0 +1,8 @@
+package errs
```

Comment:
Should we name this package as `enumer` so that it gets imported as `github.com/dmarkham/enumer/enumer` and makes it more clear when referencing the error: `enumer.ErrValueInvalid`. `errs` as a package name feels a bit like a more generic errors package

Reply:
I like the /enumer(rs)?/ pun. I'll switch to this.